sphinx_gallery/gen_rst.py

```diff
@@ -865,6 +865,20 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf,
 
     logger.debug("%s ran in : %.2g seconds\n", src_file, time_elapsed)
 
+    # Create dummy images
+    if not executable:
```

Comment:
Okay it looks like the documentation above that I just complained about might just be incorrect -- it's not just in the `not plot_gallery` condition but also excludes, etc.